### PR TITLE
You can now keep donk pockets warm with baggies!

### PR DESCRIPTION
**Pull request by ghost**
🆑 More Robust Than You & ChuckTheSheep
spriteadd: New donk pocket box sprites!
add: Warm donk pockets will now cool after around 7.5 minutes
add: Donk pockets can be kept warm for 25 mintues if kept in a sealed baggie!
/🆑 

Sadly #35600 got ruined, so i'll make due

Sprited by @ChuckTheSheep

### CODE REVIEW

**Review by WJohn — DISMISSED** [review]

**Comment by WJohn** [comment]
Sprites look good, but frankly they should be kept warm if they're kept in any container (your backpack, a box, whatever). So long as they're not kept out in the open, in your hands, or in your pockets.

File: tgstation.dme [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// DM Environment file for tgstation.dme.
+// DM Environment file for tgstation.dme./
```
Comment (ShizCalev) [inline tgstation.dme]: wut

Comment (MoreRobustThanYou) [reply tgstation.dme]: sorry, i did that to make the branch initially

**Comment by Xhuis** [comment]
where do you get the donk bags?

**Comment by MoreRobustThanYou** [comment]
WILL add that later

**Comment by ChuckTheSheep** [comment]
Make the larger full ones spawn in the freezers, and make it so the box contains like 4 individually bagged ones. If the microwave could handle it, you should be able to microwave them while in the bag?

**Comment by Ordonis** [comment]
They should also be kept warm when implanted in someone. Store warm donk pockets in clown, tear open for instant snack, steal shoes and slip the clown on his own pda.

**Comment by epochayur** [comment]
the real question is why donk pockets don't come out of the microwave at fire/lava temps and you have to wait two minutes to eat it in order to not get burn damage

**Comment by KorPhaeron** [comment]
Keeps evading and triggering the stickyban on server and causing a big mess with false positives and his PRs are the only thing he might actually care enough about to get him to stop

So his PRs are closed

**Comment by ChuckTheSheep** [comment]
!

**Comment by MoreRobustThanYou** [comment]
kor is overreacting

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 2]
```diff
@@ -189,6 +189,28 @@
 	list_reagents = list("nutriment" = 4, "omnizine" = 3)
 	tastes = list("meat" = 2, "dough" = 2, "laziness" = 1)
 	foodtype = GRAIN
+	var/cooked_time = 0
+
+/obj/item/reagent_containers/food/snacks/donkpocket/warm/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
```
Comment (vuonojenmustaturska) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: Make this a timer or something

**Review by vuonojenmustaturska — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/food/snacks_pastry.dm (hunk 2 above)
Comment (vuonojenmustaturska) [reply code/modules/food_and_drinks/food/snacks_pastry.dm]: alternatively, make the bagged ones not process() but make the heated ones get cold sooner

Comment (MoreRobustThanYou) [reply code/modules/food_and_drinks/food/snacks_pastry.dm]: this has a time that is different based on loc, not just 1 static time. also, probs

**Pull request closed without merging**